### DISCUSSION TITLE
Setting default Paypal Client ID/Secret for Organizer

### DISCUSSION THREAD

**Discussion by falaca** [pretix/pretix#4202] [Q&A] [answered]
Hi,

Is it possible to set a default Paypal Client ID and Secret for an Organizer, which could be automatically set when a new event is created and Paypal is added as a payment method? And would it be possible to hide the Secret value so that it doesn't show up in the settings, similar to how the Paypal Secret in the Global Settings is masked with asterisks?

Thanks!

**Comment by pc-coholic** [comment, marked as answer]
While you could in theory set a default clientid and secret through the Django shell, those values will not be hidden on the event settings pages - there is no way to do this right now.

If that is something you would like to do, you'd need to override the integrated PayPal plugin and roll your own, sorry.

**Comment by falaca** [comment]
Hi, thanks for your getting back to me so quickly. Is there any way that we can use our API key in the Global Settings? Given that not everybody who will be creating events has access to the organization's Paypal credentials (this is a non-profit, so this instance is just being used within the organization, so there is just one organizer), we're trying to figure out what would be the most straightforward way to allow volunteers to create events without having to enter Paypal credentials. Alternatively, if that's not possible, would you mind pointing me towards any tips for setting the default client ID/secret via the Django shell?

Thanks again!
